### ISSUE TITLE
LADSPA plugin Segmentation fault (core dumped) with ffmpeg 

### ISSUE THREAD

**Issue by Tetsujinfr** [werman/noise-suppression-for-voice#99]
hi

I am on Ubuntu 20.4LTS, x86 PC. 
I am trying to compile and run the ladspa plugin for ffmpeg.
I did follow issue #11 to get all compiled and execute the plugin with ffmep.

The compilation did complete without any issue.

But when I execute ffmpeg, I get a hard segfault crash. I got this same error with other wav input files.

```
~/noise-suppression-for-voice/build$ ffmpeg -hide_banner -i /home/tetsfr/Downloads/nice-work.wav -af ladspa=file=librnnoise_ladspa:plugin=noise_suppressor_stereo out.wav
Guessed Channel Layout for Input Stream #0.0 : stereo
Input #0, wav, from '/home/tetsfr/Downloads/nice-work.wav':
  Metadata:
    encoder         : Fission
  Duration: 00:00:00.70, bitrate: 1412 kb/s
    Stream #0:0: Audio: pcm_s16le ([1][0][0][0] / 0x0001), 44100 Hz, stereo, s16, 1411 kb/s
Stream mapping:
  Stream #0:0 -> #0:0 (pcm_s16le (native) -> pcm_s16le (native))
Press [q] to stop, [?] for help
Segmentation fault (core dumped)
```

Did I do something wrong?

**Comment by werman** [comment]
Could you execute it under debugger:

```
gdb --args ffmpeg -hide_banner -i /home/...

(gdb) bt
```

Paste the `bt` output here.

**Comment by werman** [comment]
Released v1.01, it would likely work now, at least it works with ffmpeg for me. Also only 48 kHz sample rate should be used.
